release.md

```diff
@@ -203,7 +203,7 @@ file).
 
 ```bash
 twine upload --repository-url=https://test.pypi.org/legacy/ \
-  -u="$TEST_TWINE_USERNAME" -p="$TEST_TWINE_PASSWORD" "dist/*"
+  --password="$CIRQ_TEST_PYPI_TOKEN" "dist/*"
```

Comment:
update the other part of the readme that says what environment variables you need to define.

Consider changing to using .pypirc

Reply:
Done, thank you for the catch.  There are few more references to these
variables in the repo, but I strongly suspect those scripts / instructions
are broken.  I will clean that up in a separate PR.